Danki2/Assets/Scripts/Abilities/AbilityObjects/ProjectileObject.cs

```diff
@@ -6,6 +6,12 @@ public abstract class ProjectileObject : MonoBehaviour
     protected Actor _caster;
     protected float _speed;
     protected Action<GameObject> _collisionCallback;
+    private AudioSource _audioSource;
+
+    private void Start()
+    {
+        _audioSource = gameObject.GetComponent<AudioSource>();
+    }
```

Comment:
I don't think the sound code in this file is actually doing anything, it looks like the sound is just getting played because "Play On Awake" is selected on the audio listener. I think we'd want to be playing the sound through code though as we won't always want to just play sounds on awake. I think we could either get a working example working through code here, or just remove this and the audio listener for now

Reply:
Removed the sound.

Reply:
Also added a new card to add footstep sounds